crd/fleet/fleet-crds.yaml

```diff
@@ -4121,22 +5777,103 @@ spec:
           metadata:
             type: object
           spec:
-            description: FailoverSpec represents the parameters of a single failover
-              operation.
+            description: ReplicationSpec defines the desired state of Replication
             properties:
-              dbclusterRef:
-                description: DBClusterRef is the DBCluster name to initiate a failover.
-                  The `Failover` object must be created in the same namespace as the
-                  DBCluster that it references. This field is required for Failover.
-                type: string
-              newPrimary:
-                description: NewPrimary is the standby instance to promote as the
-                  new primary. If left empty, the system will automatically pick the
-                  best one to failover to.
-                type: string
+              dbcluster:
+                description: DBCluster is the DBCluster this Replication belongs to.
+                properties:
+                  name:
+                    description: 'Name of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names
```

Comment:
- Remove quotation marks from the beginning and the end of the sentence.
- Remove the TODO comment.